tests/groups/test_api.py

```diff
@@ -12,19 +13,27 @@ async def test_find(spawn_client, all_permissions, no_permissions):
 
     await client.db.groups.insert_many(
         [
-            {"_id": "test", "name": "testers", "permissions": all_permissions},
-            {"_id": "limited", "permissions": no_permissions},
+            {
+                "_id": "test",
+                "name": "testers",
+                "permissions": all_permissions,
+                "users": [
+                    {"administrator": True, "handle": "igboyes", "id": "igboyes"}
+                ],
+            },
+            {
+                "_id": "limited",
+                "permissions": no_permissions,
+                "users": [{"administrator": False, "handle": "bob", "id": "test"}],
+            },
```

Comment:
* The group document does not contain a `users` field. It is populated from the users collection in MongoDB.
* Don't need to worry about `users` at the find endpoint as the field is not included in the minimal model.